"make distcheck" fails on synfig-core

I started preparing 1.5.2 release, but hit the following error when running `autobuild/synfigstudio-release.sh` script (the error happens after "make distcheck" command in synfig-core) -

```
...
make[3]: Leaving directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/examples'
make[2]: Leaving directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/examples'
Making check in po
make[2]: Entering directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/po'
INTLTOOL_EXTRACT="/usr/bin/intltool-extract" XGETTEXT="/usr/bin/xgettext" srcdir=../../../po /usr/bin/intltool-update --gettext-package synfig --pot
rm -f missing notexist
srcdir=../../../po /usr/bin/intltool-update -m
if [ -r missing -o -r notexist ]; then \
  exit 1; \
fi
make[2]: Leaving directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/po'
Making check in test
make[2]: Entering directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/test'
make  angle benchmark bezier bline bone clock filesystem_path keyframe node pen reference_counter string surface_etl
make[3]: Entering directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/test'
  CXX      angle.o
  CXX      benchmark.o
  CXX      hermite.o
  CXX      bline.o
../../../test/hermite.cpp:29:10: fatal error: test_base.h: No such file or directory
 #include "test_base.h"
          ^~~~~~~~~~~~~
compilation terminated.
../../../test/angle.cpp:30:10: fatal error: test_base.h: No such file or directory
 #include "test_base.h"
          ^~~~~~~~~~~~~
compilation terminated.
Makefile:789: recipe for target 'hermite.o' failed
make[3]: *** [hermite.o] Error 1
make[3]: *** Waiting for unfinished jobs....
Makefile:789: recipe for target 'angle.o' failed
make[3]: *** [angle.o] Error 1
../../../test/bline.cpp:36:10: fatal error: test_base.h: No such file or directory
 #include "test_base.h"
          ^~~~~~~~~~~~~
compilation terminated.
Makefile:789: recipe for target 'bline.o' failed
make[3]: *** [bline.o] Error 1
make[3]: Leaving directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/test'
Makefile:1148: recipe for target 'check-am' failed
make[2]: *** [check-am] Error 2
make[2]: Leaving directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub/test'
Makefile:687: recipe for target 'check-recursive' failed
make[1]: *** [check-recursive] Error 1
make[1]: Leaving directory '/home/konstantin/sources/synfig/_release/synfig-core/synfig-1.5.1/_build/sub'
Makefile:893: recipe for target 'distcheck' failed
make: *** [distcheck] Error 1
```

--- a/synfig-core/test/Makefile.am
+++ b/synfig-core/test/Makefile.am
@@ -1,7 +1,7 @@
 MAINTAINERCLEANFILES=Makefile.in
-AM_CXXFLAGS=@CXXFLAGS@ @ETL_CFLAGS@ @SYNFIG_CFLAGS@ -I$(top_builddir) -I$(top_srcdir)/src
+AM_CXXFLAGS=@CXXFLAGS@ @ETL_CFLAGS@ @SYNFIG_CFLAGS@ -I$(top_builddir) -I$(top_srcdir)/src -I$(top_srcdir)/test
 AM_LDFLAGS = \
-	../src/synfig/libsynfig.la \
+	$(top_builddir)/src/synfig/libsynfig.la \
 	@SYNFIG_LIBS@
 
 check_PROGRAMS=$(TESTS)

--- a/synfig-studio/test/Makefile.am
+++ b/synfig-studio/test/Makefile.am
@@ -1,7 +1,7 @@
 MAINTAINERCLEANFILES=Makefile.in
 AM_CXXFLAGS=@CXXFLAGS@ @SYNFIG_CFLAGS@ -I$(top_builddir) -I$(top_srcdir)/src -I$(top_srcdir)/test
 AM_LDFLAGS = \
-	$(top_srcdir)/src/synfigapp/libsynfigapp.la \
+	$(top_builddir)/src/synfigapp/libsynfigapp.la \
 	@SYNFIG_LIBS@
 
 check_PROGRAMS=$(TESTS)

--- a/synfig-studio/test/Makefile.am
+++ b/synfig-studio/test/Makefile.am
@@ -1,7 +1,7 @@
 MAINTAINERCLEANFILES=Makefile.in
-AM_CXXFLAGS=@CXXFLAGS@ @SYNFIG_CFLAGS@ -I$(top_builddir) -I$(top_srcdir)/src
+AM_CXXFLAGS=@CXXFLAGS@ @SYNFIG_CFLAGS@ -I$(top_builddir) -I$(top_srcdir)/src -I$(top_srcdir)/test
 AM_LDFLAGS = \
-	../src/synfigapp/libsynfigapp.la \
+	$(top_srcdir)/src/synfigapp/libsynfigapp.la \
 	@SYNFIG_LIBS@
 
 check_PROGRAMS=$(TESTS)

